diff -burN kydpdict-0.5.8.orig/src/kydpconfig.cpp kydpdict-0.5.8/src/kydpconfig.cpp
--- kydpdict-0.5.8.orig/src/kydpconfig.cpp	2004-01-04 12:03:44.000000000 +0100
+++ kydpdict-0.5.8/src/kydpconfig.cpp	2004-01-04 13:59:43.000000000 +0100
@@ -23,9 +23,13 @@
     QDir dir;
     QFile f;
 
+	cfgname = getenv("HOME_ETC");
+	if (cfgname == "")
+	{
 	cfgname = getenv("HOME");
 	cfgname += "/";
 	cfgname += getenv("CONFIG_DIR");
+	}
 	cfgname += "/";
 	cfgname += ".kydpdict";
 
