diff -urN kydpdict-0.5.6.orig/src/kydpdict.cpp kydpdict-0.5.6.pl2/src/kydpdict.cpp
--- kydpdict-0.5.6.orig/src/kydpdict.cpp	Tue Jul 29 01:31:52 2003
+++ kydpdict-0.5.6.pl2/src/kydpdict.cpp	Mon Sep 29 21:08:04 2003
@@ -434,9 +434,12 @@
 
 	aboutBox.setCaption("Kydpdict");
 	aboutBox.setText(
-	    tr("This is frontend to YDP Collins dictionary.\n"
-	    "Authors: Andrzej Para , Maciej Witkowiak.\n"
-	    "Program based on ydpdict by Wojtek Kaniewski"));
+		tr("This is frontend for YDP Collins and Langenscheidt electronic dictionaries.\n"
+		"It allows to use dictionary files included in theirs under Linux and probably other UNIX-like systems.\n"
+		"You must to have original software package (which working on MS-Windows only)\n"
+		"from Young Digital Poland in order to use it - see http://www.ydp.com.pl .\n\n"
+		"Authors of this frontend: Andrzej Para , Maciej Witkowiak.\n"
+        "Program based on ydpdict by Wojtek Kaniewski."));
 	aboutBox.setIconPixmap(QPixmap(babelfish_xpm));
 	aboutBox.exec();
 
diff -urN kydpdict-0.5.6.orig/src/kydpdict_pl.ts kydpdict-0.5.6.pl2/src/kydpdict_pl.ts
--- kydpdict-0.5.6.orig/src/kydpdict_pl.ts	Tue Jul 29 01:24:45 2003
+++ kydpdict-0.5.6.pl2/src/kydpdict_pl.ts	Tue Sep 30 00:19:13 2003
@@ -42,24 +42,10 @@
         <translation>Informacje o Qt</translation>
     </message>
     <message>
-        <source>This is frontend to YDP Collins dictionary.
-Authors: Andrzej Para , Maciej Witkowiak.
-Program based on ydpdict by Wojtek Kaniewski</source>
-        <translation>To jest nakładka na słownik YDP Collins.
-Autorzy: Andrzej Para, Maciej Witkowiak.
-Program oparty na ydpdict Wojtka Kaniewskiego</translation>
-    </message>
-    <message>
         <source>Error</source>
         <translation>Błąd</translation>
     </message>
     <message>
-        <source>Kydpdict can&apos;t work with incorrect path to dictionary files.
-Good bye!</source>
-        <translation>Kydpdict nie może pracować z złą ścieżką do plików słownika.
-Do widzenia!</translation>
-    </message>
-    <message>
         <source>Ctrl+.</source>
         <comment>Options|ENG --&gt; POL</comment>
         <translation>Ctrl+.</translation>
@@ -116,6 +102,30 @@
         <source>Toggle clipboard tracking</source>
         <translation>Włącz/wyłącz śledzenie schowka</translation>
     </message>
+    <message>
+        <source>Kydpdict can&apos;t work with incorrect path to dictionary files.
+Good bye!</source>
+        <translation>Kydpdict nie może działać z niepoprawną ścieżką do plików słowników.
+Jeżeli nie wiesz skąd je wziąć, zobacz informację o programie.
+Do widzenia!</translation>
+    </message>
+    <message>
+        <source>This is frontend for YDP Collins and Langenscheidt electronic dictionaries.
+It allows to use dictionary files included in theirs under Linux and probably other UNIX-like systems.
+You must to have original software package (which working on MS-Windows only)
+from Young Digital Poland in order to use it - see http://www.ydp.com.pl .
+
+Authors of this frontend: Andrzej Para , Maciej Witkowiak.
+Program based on ydpdict by Wojtek Kaniewski.</source>
+        <translation>To jest frontend dla elektronicznych słowników YDP: Collinsa i Langenscheidta.
+Pozwala on na używanie plików zawartych w tych słownikach pod Linuksem
+i prawdopodobnie innymi uniksowymi systemami. Musisz mieć oryginalny program
+(który działa tylko pod MS-Windows) wydany przez Young Digital Poland, żeby móc
+go używać - zobacz http://www.ydp.com.pl .
+
+Autorzy frontendu: Andrzej Para , Maciej Witkowiak.
+Program bazuje na ydpdict Wojtka Kaniewskiego.</translation>
+    </message>
 </context>
 <context>
     <name>ydpConfigure</name>
@@ -168,10 +178,6 @@
         <translation>Kolor zwykłego tekstu</translation>
     </message>
     <message>
-        <source>Choose path to dictionaries</source>
-        <translation>Wybierz ścieżkę do słowników</translation>
-    </message>
-    <message>
         <source>Path to audio player</source>
         <translation>Ścieżka do odtwarzacza audio</translation>
     </message>
@@ -224,13 +230,23 @@
         <translation>Wybierz ścieżkę do niemieckich plików dźwiękowych</translation>
     </message>
     <message>
-        <source>Ignore own selection</source>
-        <translation>Ignoruj własne zaznaczenia</translation>
-    </message>
-    <message>
         <source>Font</source>
         <translation>Font</translation>
     </message>
+    <message>
+        <source>Choose path to dictionaries&apos; files taken from electronic versions
+of Collins and/or Langenscheidt&apos;s dictionaries released by
+Young Digital Poland company - see http://www.ydp.com.pl
+These dictionares are not free - it is commercial software.</source>
+        <translation>Wskaż ścieżkę do plików słowników wziętych z elektronicznych wersji
+słowników Collinsa i/lub Langenscheidta wydanych przez firmę
+Young Digital Poland - zobacz http://www.ydp.com.pl .
+Słowniki YDP nie są wolne - to jest komercyjny program.</translation>
+    </message>
+    <message>
+        <source>Ignore selection in dictionary window when clipboard tracing is enabled.</source>
+        <translation>Ignoruj zaznaczenia w oknie słownika kiedy jest właczone śledzenie schowka.</translation>
+    </message>
 </context>
 <context>
     <name>ydpDictionary</name>
diff -urN kydpdict-0.5.6.orig/src/ydpconfigure.cpp kydpdict-0.5.6.pl2/src/ydpconfigure.cpp
--- kydpdict-0.5.6.orig/src/ydpconfigure.cpp	Sat Jul 26 23:25:44 2003
+++ kydpdict-0.5.6.pl2/src/ydpconfigure.cpp	Tue Sep 30 00:16:31 2003
@@ -130,7 +130,8 @@
     tab3 = new QWidget( TabWidget1, "tab3" );
     checkBox2 = new QCheckBox(tr("Enable clipboard tracking"), tab3, "checkBox2");
     checkBox2->setGeometry(10,14,350,20);
-    checkBox6 = new QCheckBox(tr("Ignore own selection"), tab3, "checkBox6");
+    checkBox6 = new QCheckBox(tr("Ignore selection in dictionary window "
+								"when clipboard tracing is enabled."), tab3, "checkBox6");
     checkBox6->setGeometry(10,40,350,20);
     checkBox3 = new QCheckBox(tr("Enable italic font"), tab3, "checkBox3");
     checkBox3->setGeometry(10,66,350,20);
@@ -178,7 +179,10 @@
 {
     QString tmp;
 
-    tmp = QFileDialog::getExistingDirectory("/", this , "path", tr("Choose path to dictionaries"));
+    tmp = QFileDialog::getExistingDirectory("/", this , "path", tr("Choose path to dictionaries\' files taken from electronic versions\n"
+																"of Collins and/or Langenscheidt\'s dictionaries released by\n"
+																"Young Digital Poland company - see http://www.ydp.com.pl\n"
+																"These dictionares are not free - it is commercial software."));
     if (!tmp.isEmpty ()) dictionaryUrl->setText(tmp);
 }


diff -urN kydpdict-0.5.6.pl2/src/ydpconfigure.cpp kydpdict-0.5.6.pl3/src/ydpconfigure.cpp
--- kydpdict-0.5.6.pl2/src/ydpconfigure.cpp	Tue Sep 30 00:16:31 2003
+++ kydpdict-0.5.6.pl3/src/ydpconfigure.cpp	Wed Oct  1 01:51:27 2003
@@ -18,7 +18,7 @@
 ydpConfigure::ydpConfigure( kydpConfig *globalconfig, QWidget* parent,  const char* name, bool modal )
     : QDialog( parent, name, modal )
 {
-    setFixedSize(400,260);
+    setFixedSize(600,260);
     setCaption( tr("Configure Kydpdict") );
 
     defaults = new QPushButton( this, "defaults" );
@@ -34,7 +34,7 @@
     ok->setGeometry(210,220,81,30);
 
     TabWidget1 = new QTabWidget( this, "TabWidget1" );
-    TabWidget1->setGeometry(10,10,381,200);
+    TabWidget1->setGeometry(10,10,581,200);
 
     tab1 = new QWidget( TabWidget1, "tab1" );
 
@@ -129,16 +129,16 @@
 
     tab3 = new QWidget( TabWidget1, "tab3" );
     checkBox2 = new QCheckBox(tr("Enable clipboard tracking"), tab3, "checkBox2");
-    checkBox2->setGeometry(10,14,350,20);
+    checkBox2->setGeometry(10,14,550,20);
     checkBox6 = new QCheckBox(tr("Ignore selection in dictionary window "
 								"when clipboard tracing is enabled."), tab3, "checkBox6");
-    checkBox6->setGeometry(10,40,350,20);
+    checkBox6->setGeometry(10,40,550,20);
     checkBox3 = new QCheckBox(tr("Enable italic font"), tab3, "checkBox3");
-    checkBox3->setGeometry(10,66,350,20);
+    checkBox3->setGeometry(10,66,550,20);
     checkBox4 = new QCheckBox(tr("Enable tooltips"), tab3, "checkBox4");
-    checkBox4->setGeometry(10,92,350,20);
+    checkBox4->setGeometry(10,92,550,20);
     checkBox5 = new QCheckBox(tr("Enable autoplay"), tab3, "checkBox5");
-    checkBox5->setGeometry(10,118,350,20);
+    checkBox5->setGeometry(10,118,550,20);
 
     TabWidget1->insertTab( tab3, tr("Miscellaneous") );
 
